ztp/ztp-policy-generator/hack/common.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -e
+
+pushd "$(dirname "$0")/.." >&2
+
+function finish {
+    popd >&2
+}
+trap finish EXIT
+
+GOPATH="${GOPATH:-~/go}"
+export GOFLAGS="${GOFLAGS:-"-mod=vendor"}"
+
+export PATH=$PATH:$GOPATH/bin
+
+PLUGINPATH=kustomize/plugin/policyGenerator/v1/policygenerator/
+PLUGINBIN=PolicyGenerator
+
+getKustomize() {
```

Comment:
I don't think we have to force install Kustomize, we just need to state kustomize is requirement if policyGen will be used as kustomize plugin. In fact we currently use the policyGen in the container image without kustomize.

Reply:
We need to force-install it because the Github container that the test runs in doesn't have it.